_data/developer.yml

```diff
@@ -352,5 +365,6 @@ websites:
     - name: Zapier
       url: https://www.zapier.com
```

Comment:
Their current URL is now `https://zapier.com/`. Could you please update the `url` to reflect this?